extensions/vertx-http/deployment/src/main/resources/dev-static/css/dev-console.css

```diff
@@ -30,3 +30,98 @@
 h1 {
     margin-bottom: 0.75em;
 }
+
+.footer {
```

Comment:
I suggest to qualify all the log-related classes so we know not to mess with them, and we also know that they don't apply elsewhere.

Reply:
Ok cool, will do.